src/TableEditColumn.js

```diff
@@ -179,6 +179,7 @@ class TableEditColumn extends Component {
     let { className } = this.state;
     // put placeholder if exist
     editable.placeholder && (attr.placeholder = editable.placeholder);
+    editable.attrs && (attr = {...editable.attrs, ...attr});
```

Comment:
I think following will be simple, keep `const` and dont assign `attr` again:

```js
const attr = {
      ref: 'inputRef',
      onKeyDown: this.handleKeyPress,
      onBlur: this.handleBlur,
      ...editable.attrs
};
```

Please check 👍   thanks